e2e/kube/examples_test.go

```diff
@@ -159,9 +189,17 @@ var _ = Describe("Examples", func() {
 			It("bosh-deployment with a persistent disk example must work", func() {
 				yamlFilePath := examplesDir + "bosh-deployment/boshdeployment-with-persistent-disk.yaml"
 
+				By("Creating a secret for implicit variable")
```

Comment:
variable?

Reply:
The `bosh-deployment-persistent-disk.yaml` example uses an implicit variable to specify the storage class for the persistent disk. For the implicit variable, I am creating a secret to specify the value of the storage class. (https://github.com/cloudfoundry-incubator/cf-operator/blob/rohit/storageclass/docs/examples/bosh-deployment/boshdeployment-with-persistent-disk.yaml#L21)